src/bundle/Resources/public/scss/_content-tree.scss

```diff
@@ -8,17 +8,20 @@
     }
 
     .ez-content-tree-container {
-        display: none;
+        position: relative;
         flex: 0 0 auto;
         max-width: 0;
-        position: relative;
         background: $ez-white;
         color: inherit;
-        transition: max-width 0.2s $ez-admin-transition;
+        transform: scaleX(0);
```

Comment:
Have you checked it on all browsers? As I remember, there were problems with `scale` and FF/Safari (?)

Reply:
In this case is ok :)